 build: force rebuild build.prop

build.prop is not rebuilt if there are only
changes in PRODUCT_PROPERTY_OVERRIDES
while ideally build.prop should be rebuilt
if there are changes in it.

diff --git a/core/Makefile b/core/Makefile
index 589c460..a4ae802 100644
--- a/core/Makefile
+++ b/core/Makefile
@@ -99,6 +99,8 @@ endif
 FINAL_DEFAULT_PROPERTIES := $(call uniq-pairs-by-first-component, \
     $(FINAL_DEFAULT_PROPERTIES),=)
 
+.PHONY: $(INSTALLED_BUILD_PROP_TARGET)
+
 intermediate_system_build_prop := $(call intermediates-dir-for,ETC,system_build_prop)/build.prop
 
 $(INSTALLED_DEFAULT_PROP_TARGET): $(intermediate_system_build_prop)
